tests/triggers/fixtures/gerrit028-trigger-simple-triggers.plugins_info.yaml

```diff
@@ -0,0 +1,3 @@
+- longName: 'Gerrit Trigger'
```

Comment:
this file is only needed when targeting an older version of the plugin that the most recent one supported by JJB